pkg/api/config.go

```diff
@@ -75,7 +75,7 @@ func validatePromotionWithTagSpec(promotion *PromotionConfiguration, tagSpec *Re
 
 func validateBuildRootImageConfiguration(fieldRoot string, input *BuildRootImageConfiguration) []error {
 	if input == nil {
-		return []error{errors.New("'build_root' is required and must have image_stream_tag or project_image")}
+		return nil
```

Comment:
Why?

Reply:
This should be conditional anyway

Reply:
So today if you omit this and other jobs depend on it you'll get an error about the root not existing, which is pretty close.  Validation probably isn't the right place for this message when you don't need source for the ci-operator job definition.

Reply:
Another option would be to make this error if and only if you have project images defined.

Reply:
Updated to continue to error if you have `images`, since images implicitly pulls in `src`